src/nxdoc/web-ui/nuxeo-elements/quality-assurance.md

```diff
@@ -174,30 +174,30 @@ We recommend reading Polymer's Documentation on [WCT](https://polymer-library.po
 
 #### Quick Guide
```

Comment:
We need to rewrite the section above this, since we are no longer using Web Component Tester (see ELEMENTS-908).